arm-servicefabric/2016-09-01/examples/ClusterGetOperation_example.json

```diff
@@ -0,0 +1,167 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "resRg",
+    "clusterName": "myCluster",
+    "api-version": "2016-09-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.ServiceFabric/clusters/myCluster",
+        "location": "southcentralus",
+        "name": "myCluster",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "clusterId": "1a1acec2-8d75-424a-81af-2fb83dd1cac8",
+          "clusterCodeVersion": "5.5.149.9494",
+          "clusterState": "Ready",
+          "managementEndpoint": "https://myCluster.southcentralus.cloudapp.azure.com:19080",
+          "clusterEndpoint": "https://warp-test-winfabrp-southcentralus.trafficmanager.net/runtime/clusters/1a1acec2-8d75-424a-81af-2fb83dd1cac8",
+          "certificate": {
+            "thumbprint": "1901CF0F76AF711BEE53DF39941092BDAC8D9050",
+            "x509StoreName": "My"
+          },
+          "clientCertificateThumbprints": [],
+          "clientCertificateCommonNames": [],
+          "fabricSettings": [
+            {
+              "name": "Security",
+              "parameters": [
+                {
+                  "name": "ClusterProtectionLevel",
+                  "value": "EncryptAndSign"
+                }
+              ]
+            }
+          ],
+          "upgradeDescription": {
+            "overrideUserUpgradePolicy": false,
+            "forceRestart": true,
+            "upgradeReplicaSetCheckTimeout": "10675199.02:48:05.4775807",
+            "healthCheckWaitDuration": "00:05:00",
+            "healthCheckStableDuration": "00:05:00",
+            "healthCheckRetryTimeout": "00:45:00",
+            "upgradeTimeout": "12:00:00",
+            "upgradeDomainTimeout": "02:00:00",
```

Comment:
looks like for duration RP is using C# TimeSpan format, if that is the case please log a bug in your system to use ISO8601 duration type instead.